docs/cheatsheet/cheat_sheet_gen.clj

```diff
@@ -127,35 +133,40 @@ Version & \\cmd{processing-version} \\\\
 (defn cheat-sheet-dynamic
   []
   (let [colours (cycle ["blue2" "yellow" "purple" "green" "orange" "pink" "blue" "yellow" "grey" "red" "green" "blue" "pink" "yellow"])
-        info    (#'processing.core/sorted-category-map)
+        info    (#'docs/sorted-category-map fn-metas)
         clean   (fn [s] (.replaceAll s "&" "\\\\&"))]
-    (dorun
-     (map (fn [[cat-idx cat-info] colour]
-            (println (str "\\colouredbox{" colour "}{"))
-            (println (str "  \\section{"(clean (:name cat-info)) "}"))
-            (println "  \\begin{tabularx}{\\hsize}{lX}")
-            (when (> (count (:fns cat-info)) 0)
-              (print (str "   & \\cmd{" ))
-              (dorun (map (fn [f] (print f " ")) (:fns cat-info)))
-              (println "} \\\\"))
-            (dorun
-             (map (fn [[subcat-idx {:keys [name fns]}]]
-                    (when (> (count fns) 0)
-                      (print (str "   " (clean name) " & \\cmd{" ))
-                      (dorun (map (fn [f] (print f " ")) fns))
-                      (println "} \\\\")))
-                  (:subcategories cat-info)))
-            (println "  \\end{tabularx}")
-            (println "}"))
-          info colours))))
+    (apply str
+      (map (fn [[cat-idx cat-info] colour]
+              (str
+                "\\colouredbox{" colour "}{\n"
+                "  \\section{" (clean (:name cat-info)) "}\n"
+                "  \\begin{tabularx}{\\hsize}{lX}\n"
+                (when (> (count (:fns cat-info)) 0)
+                  (str
+                    "   & \\cmd{"
+                    (apply str (map (fn [f] (str f " ")) (:fns cat-info)))
+                    "} \\\\\n"))
+                (apply str
+                 (map (fn [[subcat-idx {:keys [name fns]}]]
+                        (when (> (count fns) 0)
+                          (str
+                            "   " (clean name) " & \\cmd{"
+                            (apply str (map (fn [f] (str f " ")) fns))
+                            "} \\\\\n")))
+                      (:subcategories cat-info)))
+                "  \\end{tabularx}\n"
+                "}\n"))
+           info colours))))
 
 
 
 (defn mk-cheat-sheet
   []
-  (println cheat-sheet-start)
-  (cheat-sheet-dynamic)
-  (println cheat-sheet-applet)
-  (println cheat-sheet-docs)
-  (println  cheat-sheet-end)
-)
+  (str
+    cheat-sheet-start
+    (cheat-sheet-dynamic)
+    cheat-sheet-applet
+    cheat-sheet-docs
+    cheat-sheet-end))
```

Comment:
I think chet-sheet-end can be removed completely. Its information is outdated and I don't think it serves any purpose.